app/server.js

```diff
@@ -162,6 +163,8 @@ DB.connect(config.DBHOST, dbAuth).then(async () => {
           );
 
           segmentationRepeater.start();
+
+          OcrManager.start();
```

Comment:
I think that having all external services into a single ENV variable is really, really wrong.

Reply:
Obviously, we are not going to fix this now.  It's a comment for another time.  Do not change this right now.

Reply:
That env variable doesn't control the services, but controls if uwazi will try to connect to a redis service. The service-related configuration is not in this env variable. We can separate the variable for the OCR, I'm not completely against it, but I think it's mostly a decision about semantics.